.github/workflows/3_LSP_unit_tests_publish.yml

```diff
@@ -0,0 +1,83 @@
+name: Unit tests for LSP and publish
```

Comment:
publish?

Reply:
Ah, I see that in `bump` there is also publish step. We should probably split that out to make it more explicit.